api/services/Slack.js

```diff
@@ -0,0 +1,34 @@
+var format = require('util').format
+var Promise = require('bluebird')
+var request = require('request')
+var post = Promise.promisify(request.post)
```

Comment:
You may like to consider using the `request-promisify` module, which combines request + bluebird.